test/fmt.c

```diff
@@ -1185,3 +1185,68 @@ int test_text2pcap(void)
 	mem_deref(mb);
 	return err;
 }
+
+
+int test_trim(void)
```

Comment:
please add name of the module to the testcase, i.e. `test_fmt_trim()`

Reply:
Done.